README.md

```diff
@@ -80,6 +88,48 @@ the `rem-base` to `10px` otherwise components proportions will not be respected.
 @import "~ovh-ui-kit/packages/oui/stylekit.less";
 ```
 
+## Develop
+
+### Run in development mode
+
+1. Clone and install `ovh-ui-angular`
+
+```bash
+git clone https://github.com/ovh-ux/ovh-ui-angular.git
+cd ovh-ui-angular
+yarn install
+yarn link
+```
+
+2. Run `ovh-ui-angular` with [ovh-ui-kit-documentation](https://github.com/ovh-ux/ovh-ui-kit-documentation)
+
+```bash
+git clone https://github.com/ovh-ux/ovh-ui-kit-documentation.git
+cd ovh-ui-kit-documentation
+yarn install
+yarn link ovh-ui-kit
```

Comment:
Missing `yarn link ovh-ui-angular` here, right?

Reply:
It concerns only `ovh-ui-kit` here, we will have the same in ovh-ui-angular :)